basic-sql-operations.md

```diff
@@ -206,71 +180,70 @@ To insert data into a table, use the `INSERT` statement:
 INSERT INTO person VALUES("1","tom","20170912");
 ```
 
-### Select data
-
-To view the data in a table, use the `SELECT` statement:
+To insert data into a table containing data, use the `INSERT` statement:
 
 {{< copyable "sql" >}}
 
 ```sql
-SELECT * FROM person;
+INSERT INTO person(id,name) VALUES("2","bob");
 ```
 
-```
-+--------+------+------------+
-| number | name | birthday   |
-+--------+------+------------+
-|      1 | tom  | 2017-09-12 |
-+--------+------+------------+
-```
-
-### Update data
-
 To update the data in a table, use the `UPDATE` statement:
 
 {{< copyable "sql" >}}
 
 ```sql
-UPDATE person SET birthday='20171010' WHERE name='tom';
+UPDATE person SET birthday='20180808' WHERE id=2;
 ```
 
+To delete the data in a table, use the `DELETE` statement:
+
 {{< copyable "sql" >}}
 
 ```sql
-SELECT * FROM person;
+DELETE FROM person WHERE id=2;
 ```
 
-```
-+--------+------+------------+
-| number | name | birthday   |
-+--------+------+------------+
-|      1 | tom  | 2017-10-10 |
-+--------+------+------------+
-```
+> **Note:**
+> 
+> UPDATE and DELETE without WHERE filter operate on the entire table.
+>
+> DQL is used to retrieve the desired data rows from a table or multiple tables, usually the core content.
```

Comment:
Similarly, this sentence doesn't look like a part of this **note**. Rather, it reads more like an introduction of the next section `query data`